timescaledb/how-to-guides/distributed-hypertables/enable-native-replication.md

```diff
@@ -25,45 +26,80 @@ Alternatively, the function
+streaming replication on each data node.
 </highlight>
 
-Once enabled, native replication happens as part of normal inserts, by
+
+Once enabled, native replication happens as part of normal inserts by
 writing each row to multiple data nodes, and therefore requires no
-additional mechanism for replicating newly inserted data. Existing
-chunks, which are not replicated according to the configured
-replication factor, needs to be re-replicated by a background job in
-order to achieve the set replication factor.
+additional mechanism for replicating newly inserted data. When
+querying, the query planner knows how to include only one replica of
+each chunk in the query plan.
+
+
+## Handling node failures
+
+When a data node fails, queries and inserts that involve the failed
+node will fail in order to ensure data consistency until the data node
+is fully available again. If the data node cannot be recovered, native
+replication allows the node to be deleted from the multi-node cluster
+without losing data:
+
+```sql
+SELECT delete_data_node('data_node_2', force => true);
+WARNING:  distributed hypertable "conditions" is under-replicated
+```
+
+Note that it is not possible to force the deletion of a data node if
+it would mean that a distributed hypertable would permanently lose
+data.
+
+Once the failed data has been removed, some data chunks will lack
+replicas, but queries and inserts should work as normal
+again. However, the multi-node cluster remains in a vulnerable state
+until all chunks that lack replicas are fully replicated again.
+
+To view the chunks that need to be replicated use the following query:
+
+```sql
+SELECT chunk_schema, chunk_name, replica_nodes, non_replica_nodes 
+FROM timescaledb_experimental.chunk_replication_status 
+WHERE hypertable_name = 'conditions' AND num_replicas < desired_num_replicas;
+     chunk_schema      |      chunk_name       | replica_nodes |     non_replica_nodes
+-----------------------+-----------------------+---------------+---------------------------
+ _timescaledb_internal | _dist_hyper_1_1_chunk | {data_node_3} | {data_node_1,data_node_2}
+ _timescaledb_internal | _dist_hyper_1_3_chunk | {data_node_1} | {data_node_2,data_node_3}
+ _timescaledb_internal | _dist_hyper_1_4_chunk | {data_node_3} | {data_node_1,data_node_2}
+(3 rows)
+```
+
+With the information from the chunk replication status view, an
+under-replicated chunk can be copied to a new node to ensure the chunk
+has sufficient number of replicas:
+
+```sql
+CALL timescaledb_experimental.copy_chunk('_timescaledb_internal._dist_hyper_1_1_chunk', 'data_node_3', 'data_node_2');
+```
 
-To view the data nodes each chunk is replicated to, the following
-query can be used:
+The chunk copying happens over several transactions and cannot be
+rolled back automatically. If the copy operation is aborted or
+terminated prematurely by the user, an operation ID for the aborted
+copy will be logged. This operation ID can later be used to cleanup
```

Comment:
```suggestion
copy is logged. This operation ID can later be used to clean up
```

Reply:
Is this instantaneously? 
If not (a batch of jobs for example): "will be" is correct.

Reply:
No, don't use future tense.